### PR TITLE
Add provisioning intro

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/docs/concepts/identity-provisioning-intro.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## What is identity provisioning? 
+
+Identity provisioning is the process of creating, updating, managing, and removing digital identities in a system or application. The main purpose of identity provisioning using an identity management system like WSO2 Identity Server (WSO2 IS), is to automate the process of creating and managing these user identities so that it can be done as efficiently as possible, even on multiple platforms or applications. 
+
+----
+
+## Why do I need provisioning?
+
+Most enterprise solutions adopt products and services from multiple cloud providers to accomplish various business requirements. This makes it insufficient to maintain user identities only in a corporate LDAP. Managing user accounts on multiple applications can be challenging and can lead to outdated information, outdated permissions, data mismatch, and security issues. Identity provisioning plays a key role in propagating user identities across different SaaS providers.
```
Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:11]: Let's use "corporate user store (corporate LDAP)"

Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:11]: WE have to say that "due to this reason we have to maintain user information in multiple places"

Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:11]: not only SaaS providers maybe we can say "across multiple systems"

File: en/docs/concepts/identity-provisioning-intro.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+For instance, if an employee leaves an organization, their user account may need to be removed and permissions may need to be revoked from multiple applications within the organization. However, it takes extra effort to ensure that it is properly removed from all applications, and doing this manually opens up the possibility of human error thereby possibly causing security issues. A much cleaner and more efficient approach would be to use identity provisioning to automate this and ensure that the account deletion or permission revoking reflects on the linked user accounts across all applications.
```
Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:13]: this does not connect this para to previous one

Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:13]: we can further say that "This is a main privacy regulatory requirement"

File: en/docs/concepts/identity-provisioning-intro.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+## How does it work?
+
+WSO2 IS can ensure that provisioning is made easy. A provisioning request can be sent to WSO2 IS to add or remove a user and this user is provisioned or deprovisioned accordingly across various applications that are configured with WSO2 Identity Server. The diagram below illustrates the process.
+
+<img name='identity-provisioning-diagram' src='../../../assets/img/concepts/identity-provisioning.png' class='img-zoomable'/>
+
+After a user has been provisioned, WSO2 IS can also help maintain the user's updated information across all configured applications. For example, to update a user attribute such as telephone number, you can update the information in one place and WSO2 IS will ensure that it is reflected in all connected user accounts.
```
Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:23]: connected applications ?

File: en/docs/concepts/identity-provisioning-intro.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+----
+
+## Types of identity provisioning
+
+Provisioning is bi-directional, hence there are two main types.
```
Comment (isharak) [inline en/docs/concepts/identity-provisioning-intro.md:29]: bi-directional provides a different meaning, it seems in the same connection we do provisioning in both sides.
maybe we can say two types

File: en/docs/concepts/identity-provisioning-intro.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# Introduction to Identity Provisioning
+
+## What is identity provisioning? 
+
+Identity provisioning is the process of creating, updating, managing, and removing digital identities in a system or application. The main purpose of identity provisioning using an identity management system like WSO2 Identity Server (WSO2 IS), is to automate the process of creating and managing these user identities so that it can be done as efficiently as possible, even on multiple platforms or applications.
```
Comment (gomathyk) [inline en/docs/concepts/identity-provisioning-intro.md:5]: Shall we divide the second sentence? It seems a bit long.

File: en/docs/concepts/identity-provisioning-intro.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+## What is identity provisioning? 
+
+Identity provisioning is the process of creating, updating, managing, and removing digital identities in a system or application. The main purpose of identity provisioning using an identity management system like WSO2 Identity Server (WSO2 IS), is to automate the process of creating and managing these user identities so that it can be done as efficiently as possible, even on multiple platforms or applications. 
+
+----
+
+## Why do I need provisioning?
+
+Most enterprise solutions adopt products and services from multiple cloud or on premise providers, to accomplish various business requirements. This means user identity information must be maintained in multiple places and it is insufficient to maintain user identities only in a corporate user store (LDAP). Managing user accounts on multiple applications can be challenging and can lead to outdated information, outdated permissions, data mismatch, and security issues. Identity provisioning plays a key role in propagating user identities across different systems. It also supports updating or removing user identities across multiple systems.
```
Comment (gomathyk) [inline en/docs/concepts/identity-provisioning-intro.md:11]: Please change "userstore" to be one word for consistency wherever applicable.

**Review by gomathyk — CHANGES_REQUESTED** [review]

**Review by gomathyk — APPROVED** [review]

**Pull request merged**